core/src/main/java/com/oath/oak/OakRValueBufferImpl.java

```diff
@@ -197,15 +195,6 @@ public double getDouble(int index) {
         throw new RuntimeException("Transform failed: reached retry limit (1024).");
     }
 
-    @Override
-    public void unsafeCopyBufferToIntArray(int srcPosition, int[] dstArray, int countInts) {
-        Slice s = getValueSlice();
-        start(s);
```

Comment:
So Liran suggests that start and end will be called by the user when necessary, and it is not supported internally

Reply:
No, to the contrary. This PR does not change this behavior.
Specifically here, I removed `unsafeCopyBufferToIntArray()` entirely. The other methods (e.g., `getInt()`) still uses start/end.